src/update.c

```diff
@@ -223,20 +262,28 @@ static void run_postupdate_action(void)
 	free(post_update_action);
 }
 
+static bool system_on_mix(void)
+{
+	return !access("/usr/share/defaults/swupd/mixed", R_OK);
```

Comment:
*Mumbles about type promotion in C...*  ;) 
I'll change the definition for clarity.